src/server/dflycmd.h

```diff
@@ -118,9 +118,11 @@ class DflyCmd {
 
   void OnClose(ConnectionContext* cntx);
 
-  // Stop all background processes so we can exit in orderly manner.
   void BreakOnShutdown();
```

Comment:
I didn't understand what this is for

Reply:
if we receive sigterm/sigint - to stop the processes.

Reply:
I've added Shutdown for this.

BreakOnShutdown is somehow linked to the connection context, but we have OnClose for this....

Reply:
OnClose is for a connection close. 
BreakOnShutdown is for the process shutdown and is more forceful.  
BreakOnShutdown should not be linked to connection context.

Reply:
No, its not. BreakOnShutdown is not a process global stop handler. You can tell just by looking its registered at `Service::CreateContext` and its called on loosing a connection.

That's way I've added `Shutdown`, so I'll keep this empty for now

Reply:
you are right. afaik, BreakOnShutdown cancel long-running, potentially never-ending operations. 
For transaction, it's blocking commands.